deploy_snapshot.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+SLUG="braintree/android-card-form"
```

Comment:
Is this supposed to be `browser-switch-android`?

Reply:
😅    yup haha. Missed this one in the copy paste. There was no `deploy_snapshot.sh` script originally.